spec/features/idv/doc_auth/address_step_spec.rb

```diff
@@ -56,7 +56,10 @@
   it 'allows the user to enter in a new address' do
     expect_step_indicator_current_step(t('step_indicator.flows.idv.verify_info'))
     expect(page).not_to have_content(t('forms.example'))
-    fill_out_address_form_ok
+    fill_in 'idv_form_address1', with: '123 Main St'
+    fill_in 'idv_form_city', with: 'Nowhere'
+    select 'Virginia', from: 'idv_form_state'
+    fill_in 'idv_form_zipcode', with: '66044'
```

Comment:
It looks like you used fill_in and select so that you could delete `fill_out_address_form_ok` in doc_auth_helper because the doc_auth_helper method is not being used anywhere else. Also, we have this same method in in_person_helper. 

I wondered if we could remove include DocAuthHelper at top of file- looks like yes but only because IdvStepHelper includes it. I think I like it being in the include list. 

Feel free to resolve this comment if this is accurate. Just wanted to confirm. I originally didn't like the change because it is more line of code but I like it now that I understand why I think you did it.

Reply:
Correct, I didn't think it made sense to have a helper method in doc_auth_helper that was only being used in one place.  I also thought having two methods with the same name could be a source of confusion.

I will try removing `DocAuthHelper` from the imports.

Reply:
I agree with it being a source of confusion. Good clean up and thanks for confirming my understanding was accurate. Resolving thread